srcpkgs/gmic/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'gmic'
 pkgname=gmic
 version=2.1.8
-revision=4
+revision=6
```

Comment:
what's the reason for skipping one number?

Reply:
I can't count